web/packages/teleterm/src/ui/Search/useSearch.ts

```diff
@@ -114,6 +114,10 @@ export function useFilterSearch() {
               .includes(search.toLocaleLowerCase())
           );
         }
+        // Cluster filter should not be visible if there is only one cluster
+        if (clusters.length === 1) {
+          return [];
+        }
```

Comment:
This makes it so that we don't show any filters if your search term limits the list of clusters to just one cluster.

Could you work on that tomorrow and add a test case for this?


https://user-images.githubusercontent.com/27113/231152357-404c8baa-40bb-42bb-9899-52eb1360aab6.mov